File: 240304/밭의 높이를 고르게하기/equalizing-the-height-of-the-field.py
import sys
n,h,t=map(int,input().split())
height=list(map(int,input().split()))

ans=sys.maxsize
for i in range(n):
    for j in range(i+1,n):
        if j-i+1==t:
            cnt=0
            for k in range(i,j+1):
                cnt+=abs(h-height[k])
            ans=min(ans,cnt)


print(ans)

File: 260110/밭의 높이를 고르게하기/equalizing-the-height-of-the-field.js
const fs = require("fs");
const input = fs.readFileSync(0).toString().trim().split('\n');
const [n,h,t]= input[0].split(' ').map(Number);
const arr = input[1].split(' ').map(Number);

let ans = Number.MAX_SAFE_INTEGER;
for(let i=0; i<n-t+1; i++){
    let cost = 0;
    for(let j=i; j<i+t; j++){
        cost += Math.abs(h-arr[j]);
    }
    ans = Math.min(cost,ans);
}

console.log(ans)